rewrote the intro (#29)
MrLongbottom/P9-Rapport: sections/introduction.tex

\vejleder[inline]{should be twice as long as it is now}
%% Argue briefly for the relevance of the studied area/problem — start from general and end with your concrete problem.

Many news articles are produced every day and the need for searching the news is becoming a more prominent and difficult task.
A general challenge for search engines, no matter the environment, is how to acquire relevant search results from a given corpus. 
While some engines use simple algorithms, such as finding all articles that include a given search term and sorting by date, this may not always give useful results.

> Information retrieval within these search engines apply many different techniques for either ranking or filtering documents\cite{google_pagerank2006}.

Author: ```suggestion
Information retrieval within search engines apply many different techniques for either ranking or filtering documents\cite{google_pagerank2006}.
```

\gls{lda} uses Dirichlet distributions to calculate the probability of a word belonging to a given topic and a document containing a given topic.
Topic modeling does not rank documents, which is why an additional part is necessary if we want to search the documents.

Ranking articles can be done in multiple ways, where PageRank is commonly used\cite{google_pagerank2006}.
PageRank\cite{pagerank_1999} is a ranking algorithm, which is based on the random walk process.
The principle is to let a number of surfers walk between nodes in a graph, where each surfer takes one step in each iteration.
After an appropriate number of steps, the number of surfers on each node will be the ranking of the given node, where more surfers give a higher rank.
The edges in the graph can be based on many interactions, such as hyperlinks, similarities, genres, etc.

In \cite{yang2009topic} they investigate how to combine topic model \gls{lda} and PageRank.
They also investigate a modified version of the PageRank algorithm which takes topic-level information into account.

> They use a scientific article dataset where they manually label their dataset with queries. 

Reviewer: Labelede de artiklerne med queries?

Author: ```suggestion
They use a scientific article dataset where they manually label 200 articles from their dataset based on related queries. 
```

> They use a scientific article dataset where they manually label their data set with queries. 

Reviewer: ```suggestion
They use a scientific article dataset where they manually label their dataset with queries. 
```

> They use an scientific article data set where they manually label their data set with queries. 

Reviewer: ```suggestion
They use a scientific article dataset where they manually label their data set with queries. 
```

> They use an scientific article data set where they manually label their data set with queries. 

Reviewer: Labelede de artiklerne med queries?

In this paper, we want to expand the framework which \cite{yang2009topic} have provided.

> But the data set which \cite{yang2009topic} uses is based on scientific articles, which includes citation to other similar articles.

Reviewer: ```suggestion
However, the data set which \cite{yang2009topic} uses is based on scientific articles, which includes citation to other similar articles.
```

> However, the citation scheme does not extent to other document types, such as news articles.

Reviewer: ```suggestion
This citation scheme does not necessarily extent to other document types, such as news articles.
```

> However, the citation scheme does not extent to other areas, such as news articles data sets.

Reviewer 2: ```suggestion
However, the citation scheme does not extent to other document types, such as news articles.
```

> In this paper, we want to investigate whether using a similarity measure based on the topics generated by a topic model can be used to PageRank.

Reviewer: ```suggestion
In this paper, we want to investigate whether using a similarity measure based on the topics generated by a topic model can be used with PageRank.
```

The combination of \gls{lda} and PageRank is interesting, since underlying document topics can yield more information than usually given by a single document.
These topics can be seen as a set of documents sharing a specific topic. 

When evaluating search engines, manual labeling of documents is usually used to measure the performance of models\cite{yang2009topic}\cite{Tang2008}.

> Since the topics, that is created by the \gls{lda}, is a distribution between of topics over documents, we can use these to evaluate a search algorithm's retrieval performance. 

Reviewer: ```suggestion
Since the topics, that are created by the \gls{lda}, are distributions of topics in documents, we can use these to evaluate a search algorithm's retrieval performance. 
```

Our goal is thus to improve news article search by using a combination of \gls{lda} and PageRank.
For our experiments we use the Nordjyske database of news articles as our dataset.

> We also want to investigate whether a automatic evaluation measure can be used to evaluate a search engine. 

Reviewer: ```suggestion
We also want to investigate whether an automatic evaluation measure can be used to evaluate a search engine. 
```
